src/public/common/Table/sortingAndFiltering.js

```diff
@@ -0,0 +1,47 @@
+import _ from 'lodash';
+
+import { safeToString } from '../../../utils/global';
+
+function ascendingComparator(a, b, sortBy) {
+  if (a[sortBy] === undefined || a[sortBy] < b[sortBy]) return -1;
+  if (a[sortBy] === undefined || a[sortBy] > b[sortBy]) return 1;
+  return 0;
+}
+
+export function getComparator(columns, order, sortBy) {
+  const column = columns.find(col => col.id === sortBy);
+  const columnComparator = column?.comparator
+    ? column.comparator
+    : ascendingComparator;
+
+  return order === 'asc'
+    ? (a, b) => columnComparator(a, b, sortBy)
+    : (a, b) => -columnComparator(a, b, sortBy);
+}
+
+export function stableSort(array, comparator) {
```

Comment:
The sort in the standard library would work as well. It is ok to remove this sorting function

Reply:
Done!